typings/index.d.ts

```diff
@@ -362,7 +362,6 @@ declare module 'discord.js' {
     Events: {
```

Comment:
Not directly related to these changes, but why is `guildMemberAvailable` [not documented](https://discord.js.org/#/docs/main/stable/class/Client)? It is part of `ClientEvents` and emitted in `PresenceUpdate.js`.

Reply:
That event eas long removed iirc.. might need to update things

Reply:
Only the docstring was removed in f3cad81f5314a3997ac188277e2471ba84cbd771, the code emitting it was not touched. This looks more like an oversight, than an intended removal.